### PR TITLE
temprature value fecthed

### PR DESCRIPTION
**Pull request by Vutsuak16**
currently printing to console

### CODE REVIEW

File: Components/ATMega_PIN_Diagram.py [hunk 1]
```diff
@@ -139,6 +139,8 @@ def __init__(self):
             self.temperatureFrame.layout = QVBoxLayout()
             self.temperatureFrame.layout.setAlignment(Qt.AlignLeft)
             self.temperatureFrame.layout.addStretch()
+            
+            self.verticalSlider.valueChanged[int].connect(self.changeValue)
 
 
             for val in temperature:
```
Comment (minallaad) [inline Components/ATMega_PIN_Diagram.py]: What is the role of this loop?

Comment (Vutsuak16) [reply Components/ATMega_PIN_Diagram.py]: it detects if the value is changed then passes it on to the helper function.
thats how you fetch widget values in pyqt5
https://www.riverbankcomputing.com/static/Docs/PyQt5/signals_slots.html

Comment (minallaad) [reply Components/ATMega_PIN_Diagram.py]: I was talking about this loop :   for val in temperature:

Comment (Vutsuak16) [reply Components/ATMega_PIN_Diagram.py]: I didn't write that code haha

Comment (minallaad) [reply Components/ATMega_PIN_Diagram.py]: If there is no logic under for loop so can you just remove the for loop?

**Pull request merged**